### PR TITLE
Subsquid based crawler

### PR DESCRIPTION
**Pull request by chernetsky**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -4,9 +4,16 @@ POSTGRES_PASSWORD=polkastats
 POSTGRES_DATABASE=polkastats
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
-LOGGING=1
-WS_PROVIDER_URL=wss://ws-rc.unique.network
-TYPE_PROVIDER=quartz
+LOGGING=false
```
Comment (sswebcoder) [inline .env:7]: In TypeORM config we waiting number for define logging level

https://github.com/UniqueNetwork/unique-scan-backend/pull/40/files#diff-9af894d792453ec3d2a15d629d9731b3cd7d17d7de45b7f0cf64b671d9246c15R42

Maybe better to use enum with names of logging levels like - info, debug, warning, error. But it's not the task for this PR.

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,4 @@ lerna-debug.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.env
```
Comment (sswebcoder) [inline .gitignore:36]: If you add file to gitignore git will not detect changes in that file but that file will be exist in repository. If you want to remove file from git, you should remove file, commit that change and after it add that file to gitignore.

Only if I right...

File: apps/crawler/src/crawler.module.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import typeormConfig from '@common/typeorm.config';
+// import { Collections } from '@entities/Collections';
```
Comment (sswebcoder) [inline apps/crawler/src/crawler.module.ts:2]: Why do you save this code in comments if you moved entities to config file?

File: apps/crawler/src/crawler.service.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+@Injectable()
+export class CrawlerService {
+  constructor(
+    private logger: Logger,
+    private dataSource: DataSource,
+    private collectionsProcessor: CollectionsProcessor,
+    private tokensProcessor: TokensProcessor,
+  ) {
+    this.logger = new Logger('CrawlerService');
```
Comment (sswebcoder) [inline apps/crawler/src/crawler.service.ts:16]: Another way without string literals. https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging

File: .env (hunk 1 above)
Comment (chernetsky) [reply .env:7]: As they say "moped is not mine")) Ok, I will change value to 0.

File: .gitignore (hunk 2 above)
Comment (chernetsky) [reply .gitignore:36]: Fixed. You were right.

File: apps/crawler/src/crawler.module.ts (hunk 3 above)
Comment (chernetsky) [reply apps/crawler/src/crawler.module.ts:2]: Just forgot to clean up. Fixed.

File: apps/crawler/src/crawler.service.ts (hunk 4 above)
Comment (chernetsky) [reply apps/crawler/src/crawler.service.ts:16]: I knew there is a more feng shui way! Thanks!

File: apps/crawler/src/main.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { NestFactory } from '@nestjs/core';
+import { CrawlerModule } from './crawler.module';
+import { CrawlerService } from './crawler.service';
+
+async function bootstrap() {
+  const app = await NestFactory.create(CrawlerModule, {
+    logger: ['log', 'error', 'warn', 'verbose'],
+  });
+
+  const crawlerService = app.get(CrawlerService);
```
Comment (sswebcoder) [inline apps/crawler/src/main.ts:10]: Why you do it in this place but not in the crawler module constructor?

**Pull request merged**